openstack/compute/v2/servers/requests.go

```diff
@@ -183,9 +183,18 @@ type CreateOpts struct {
 	// AccessIPv4 specifies an IPv4 address for the instance.
 	AccessIPv4 string `json:"accessIPv4,omitempty"`
 
-	// AccessIPv6 pecifies an IPv6 address for the instance.
+	// AccessIPv6 specifies an IPv6 address for the instance.
 	AccessIPv6 string `json:"accessIPv6,omitempty"`
 
+	// Min specifies Minimum number of servers to launch.
+	Min int `json:"min_count,omitempty"`
+
+	// Max specifies Maximum number of servers to launch.
+	Max int `json:"max_count,omitempty"`
+
+	// ReturnReservationID sets respond with reservation_id only.
+	ReturnReservationID bool `json:"return_reservation_id,omitempty"`
```

Comment:
If I understand this correctly, setting this to `true` will cause the response body to be a different format? If so, then this will cause a JSON error when running `Create` and `Extract`. To get around this, we will need to create a new `Extract` method to handle this. This seems to be out of scope for the original issue this PR is fixing, so let's remove this.

Reply:
right. just removed that code. let's do it when it is needed.